ui/pages/Report/components/SampleMetadata.jsx

```diff
@@ -4,13 +4,19 @@ import { connect } from 'react-redux'
 
 import FormWrapper from 'shared/components/form/FormWrapper'
 import { BaseSemanticInput } from 'shared/components/form/Inputs'
+import { ALL_PROJECTS_PATH, GREGOR_PROJECT_PATH } from '../constants'
 import { loadSampleMetadata } from '../reducers'
 import { getSampleMetadataLoading, getSampleMetadataLoadingError, getSampleMetadataRows, getSampleMetadataColumns } from '../selectors'
 import BaseReport from './BaseReport'
 
+const FILENAME_LOOKUP = {
+  [ALL_PROJECTS_PATH]: 'All_AnVIL_Projects',
+  [GREGOR_PROJECT_PATH]: 'All_GREGoR_Projects',
+}
+
 const getDownloadFilename = (projectGuid, data) => {
-  const projectName = projectGuid && projectGuid !== 'all' && data.length && data[0].project_id.replace(/ /g, '_')
-  return `${projectName || 'All_AnVIL_Projects'}_${new Date().toISOString().slice(0, 10)}_Metadata`
+  const projectName = FILENAME_LOOKUP[projectGuid] || (data.length && data[0].project_id.replace(/ /g, '_'))
```

Comment:
The behavior changed slightly when `projectGuid` is None or '' the `projectName` changes from 'All_AnVIL_Projects' to be generated from the `data`.

Reply:
That case is only reachable before the user has actually loaded any data, so they would not be downloading the report anyways